src/anaconda/.github/dependabot.yml

```diff
@@ -3,10 +3,10 @@
 # Please see the documentation for more information:
 # https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
 # https://containers.dev/guide/dependabot
```

Comment:
Looping in @joshspicer for another set of eyes.

If we decide to merge this, you'd need to bump patch versions for all the updated Templates.

eg. https://github.com/devcontainers/templates/blob/791efeddf955f451e40894e7bc0397578f56ef11/src/anaconda/devcontainer-template.json#L3